azure-pipelines.yml

```diff
@@ -23,7 +23,21 @@ pr:
 - production
 
 stages:
+- stage: GitHub_Issue_Verification
```

Comment:
Can this be part of the build step, so it doesn't require a whole separate machine allocation? Those aren't cheap and if things are getting heavy usage, can add half an hour to a build.

Reply:
I think it's better as a stage so that you can rerun it independent of your build if you forgot to add the necessary information. It also needs to run on the hosted agents so that it runs quickly enough to be useful.

Reply:
I could have been a GitHub Action?